examples/dpu.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: config.openshift.io/v1
```

Comment:
---
apiVersion: v1
kind: Namespace
metadata:
  name: dpu-operator-system
--- 
<rest of yaml for each file>

Reply:
dpu-operator-system already created by operator sdk